src/main/webapp/WEB-INF/app/controllers/settings/organizationManagementController.js

```diff
@@ -61,6 +61,22 @@ vireo.controller("OrganizationManagementController", function ($controller, $q,
 		$scope.cancelDeleteOrganization = function() {
 			$scope.closeModal();
 			$scope.getSelectedOrganization().clearValidationResults();
+    };
+    
+    $scope.restoreOrganizationDefaults = function(organization) {
+			organization.restoreDefaults().then(function(data) {
+				if(data.meta.type != 'INVALID') {
+					$scope.closeModal();
+					$timeout(function() {
+						AlertService.add(data.meta, 'organization/restore-defaults');
+					}, 300);
+				}
+			});
+		};
+
+		$scope.cancelDeleteOrganization = function() {
```

Comment:
This is a duplicate function name. Probably should run grunt to perform linting. This is also why the modal is not closing on cancel.